src/Pages/Form/Form.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useState } from 'react';
+import "./Form.css";
+import Modal from '../../Components/Modal/Modal';
+
+import openIcon from "../../assets/dot-menu.png";
+import editIcon from "../../assets/edit_icon.png";
+import saveIcon from "../../assets/save_icon.png";
+
+const TextboxWithOverlay = () => {
+
+
+  const [form, setForm] = useState({
+    Compagny_name: "",
+    Compagny_address: "",
+    Compagny_website: "",
+    Compagny_phone_number: "",
+    Manufacturer_name: "",
+    Manufacturer_address: "",
+    Manufacturer_website: "",
+    Manufacturer_phone_number: "",
+    Fertiliser_name: "",
+    Fertiliser_registration_number: "",
+    Fertiliser_lot_number: "",
+    Fertiliser_NPK: "",
+    Fertiliser_precautionary_FR: "",
+    Fertiliser_precautionary_EN: "",
+    Fertiliser_instructions_FR: "",
+    Fertiliser_instructions_EN: "",
+    Fertiliser_ingredients_FR: "",
+    Fertiliser_ingredients_EN: "",
+    Fertiliser_specifications_FR: "",
+    Fertiliser_specifications_EN: "",
+    Fertiliser_cautions_FR: "",
+    Fertiliser_cautions_EN: "",
+    Fertiliser_recommendation_FR: "",
+    Fertiliser_recommendation_EN: "",
+    Fertiliser_first_aid_FR: "",
+    Fertiliser_first_aid_EN: "",
+    Fertiliser_warranty_FR: "",
+    Fertiliser_warranty_EN: "",
+    Fertiliser_danger_FR: "",
+    Fertiliser_danger_EN: "",
+    Fertiliser_guaranteed_analysis: "",
+    Nutrient_in_guaranteed_analysis: "",
+    Percentage_in_guaranteed_analysis: "",
+    Fertiliser_weight: "",
+    Fertiliser_density: "",
+    Fertiliser_volume: "",
+    Fertiliser_label_all_other_text_FR: "",
+    all_other_text_FR_1: "",
+    all_other_text_FR_2: "",
+    Fertiliser_label_all_other_text_EN: "",
+    all_other_text_EN_1: "",
```

Comment:
This is generative variable. On text can produce only all_other_text_EN_1 and the other text would produce only all_other_text_EN_1, only all_other_text_EN_2, only all_other_text_EN_3, only all_other_text_EN_4, only all_other_text_EN_5 etc so make sure to make it adaptive to the number of value given by the API.